packages/amp/src/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { CacheProvider } from "frontity";
+import { renderToStaticMarkup } from "react-dom/server";
+import createEmotionServer from "@emotion/server/create-instance";
+import createCache from "@emotion/cache";
+import ampTemplate from "./template";
+import AMP from "../types";
+
+/**
+ * Emotion cache key.
+ */
+const CACHE_KEY = "css-amp";
+
+/**
+ * Remove a final `/amp` from a link.
+ *
+ * @param link - The link to be processed.
+ *
+ * @returns The same link but without the final `/amp`.
+ */
+export const removeAmp = (link: string): string =>
+  link.replace(/\/amp\/?($|\?|#)/, "/");
+
+export default {
+  name: "@frontity/amp",
+  actions: {
+    amp: {
+      init: ({ libraries }) => {
+        if (libraries.source) {
```

Comment:
@michalczaplinski I've added this check in here, since there can be static website without a _source_. And those kind of website do not need this parsing. Am I assuming correct in here?